4.analyze_data/notebooks/UMAP/scripts/0.UMAP_coordinates.py

```diff
@@ -69,19 +69,29 @@
 # Fit UMAP features per dataset and save
 for plate in cp_dfs:
     plate_name = pathlib.Path(plate).stem
-    # Make sure to reinitialize UMAP instance per plate
+    print("UMAP embeddings being generated for", plate_name)
+
+     # Make sure to reinitialize UMAP instance per plate
     umap_fit = umap.UMAP(
         random_state=umap_random_seed,
         n_components=umap_n_components
     )
-    
+
     # Make sure NA columns have been removed
     cp_df = cp_dfs[plate]
     cp_df = feature_select(
         cp_df,
         operation="drop_na_columns",
         na_cutoff=0
     )
+
+    # Make sure that the Plate_3_prime has correct name in Metadata_Plate column
+    if plate_name.replace("_sc_feature_selected", "") == "Plate_3_prime":
+        cp_df["Metadata_Plate"] = "Plate_3_prime"
```

Comment:
Does the plate not already have the correct name?

Reply:
I don't think so. I do believe it is a current issue with the regular expression in the CellProfiler pipeline, so this is added as a just in case too.